cookbooks/aws-parallelcluster-install/recipes/install_mysql_client.rb

```diff
@@ -15,55 +15,48 @@
 # OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions and
 # limitations under the License.
 
-def install_repository_configuration_package(repository_name, definition_source, local_file, md5_hash)
-  remote_file local_file do
-    source definition_source
-    mode '0644'
+if platform?('ubuntu') && arm_instance?
+  package node['cluster']['mysql']['repository']['packages'] do
     retries 3
     retry_delay 5
-    not_if { ::File.exist?(local_file) }
   end
+else
 
-  ruby_block "Validate Repository Definition Checksum" do
-    block do
-      require 'digest'
-      checksum = Digest::MD5.file(local_file).hexdigest
+  package_installer = value_for_platform(
+    'default' => "yum install -y",
+    'ubuntu' => { 'default' => "apt install -y" }
+  )
 
-      if checksum != md5_hash
-        raise "Downloaded file #{local_file} checksum #{checksum} does not match expected checksum #{md5_hash}"
-      end
-    end
-  end
+  mysql_archive_url = node['cluster']['mysql']['package']['archive']
+  mysql_tar_file = "/tmp/#{node['cluster']['mysql']['package']['file-name']}"
 
-  if platform?('ubuntu')
-    # dpkg_package is used here because 'package' seems to default to using apt_package
-    # which fails on the MySQL package.
-    dpkg_package repository_name do
-      source local_file
-    end
+  remote_file mysql_tar_file do
+    source mysql_archive_url
+    mode '0644'
+    retries 3
+    retry_delay 5
+    not_if { ::File.exist?(mysql_tar_file) }
+  end
```

Comment:
[minor] Could we please put a ruby expression to log the `mysql_archive_url` in the Cinc logs? At the moment this is not logged by the `remote_file` resource.